packages/cli/src/logger/index.ts

```diff
@@ -551,9 +551,9 @@ export const logHook = (hook = '', msg = '') => {
     console.log(`${`[hook]`} ${_sanitizePaths(msg)}`);
 };
 
-export const logWarning = (msg: string | boolean | unknown) => {
+export const logWarning = (msg: string | boolean | unknown, shouldSanitizePath = false) => {
```

Comment:
It seems like shouldSanitizePath is backwards here? And using an options object like
```
export const logWarning = (msg: string | boolean | unknown, opts?: { skipSanitizePaths?: boolean }) => {
```
would make the usage clearer.